daemon/src/peer/run_state.rs

```diff
@@ -26,12 +26,17 @@ use librad::{
     },
     peer::PeerId,
 };
+use waiting_room::TransitionWithResult;
 
 use crate::{
     convert::MaybeFrom,
     peer::{announcement, control},
-    request::waiting_room::{self, WaitingRoom},
+    request::{
+        waiting_room::{self, Transition as WaitingRoomTransition, WaitingRoom},
```

Comment:
I'm more fond of using crate prefixes instead of renaming, so `waiting_room::Transition` rather than `WaitingRoomTransition`. I find it more clear when tracking through types.

What do you think?